Tasks/day.js

```diff
@@ -1,15 +1,13 @@
 // Get day number
+'use strict';
 
-let D = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+const Week = ['sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat'];
```

Comment:
```suggestion
const DAYS = ['sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat'];
```
or `WEEK` or `WEEK_DAYS` or `DAYS_OF_WEEK`